frontend/src/components/opportunity/OpportunityCTA.tsx

```diff
@@ -0,0 +1,58 @@
+import { OpportunityStatus } from "src/types/opportunity/opportunityResponseTypes";
+
+import { useTranslations } from "next-intl";
+import { ReactNode } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+export const OpportunityContentBox = ({
+  title,
+  content,
+}: {
+  title?: string | ReactNode;
+  content: string | ReactNode;
+}) => {
+  return (
+    <div className="border radius-md border-base-lighter padding-x-2">
+      {title && <p className="text-bold margin-bottom-0">{title}</p>}
+      <p className="desktop-lg:font-sans-sm margin-top-0">{content}</p>
+    </div>
+  );
+};
+
+const OpportunityCTA = ({
+  status,
+  id,
+}: {
+  status: OpportunityStatus;
+  id: number;
+}) => {
+  const t = useTranslations("OpportunityListing.cta");
+
+  const contentKey = status === "posted" ? "apply_content" : "closed_content";
+  const externalOpportunity = `https://grants.gov/search-results-detail/${id}`;
```

Comment:
No idea what the level of effort would be to configure this, but we might want to have this change depending on the environment.

In prod mapping the grants.gov is fine, but it would be confusing in our lower environments as the links might work (if that ID happens to also exist) but the opportunity would clearly be different. We'd want `test.grants.gov/...` for that.

Reply:
That's a good idea, and definitely doable. So, to double check since I don't have a full understanding of our non-prod environments, would the logic look like this?
```
env == prod -> https://grants.gov/
env != prod -> https://test.grants.gov/
```